README.md

```diff
@@ -1,31 +1,36 @@
 # jooq-testcontainers-codegen-maven-plugin
 
 The `jooq-testcontainers-codegen-maven-plugin` simplifies the jOOQ code generation 
-by using [Testcontainers](https://www.testcontainers.org/) and applying Flyway database migrations.
+by using [Testcontainers](https://www.testcontainers.org/) and applying database migrations.
 
 [![Build](https://github.com/testcontainers/jooq-testcontainers-codegen-maven-plugin/actions/workflows/build.yml/badge.svg)](https://github.com/testcontainers/jooq-testcontainers-codegen-maven-plugin/actions/workflows/build.yml)
 
 ## Supported databases:
 * Postgres
 * MySQL
 * MariaDB
+* 
+## Supported migration tools:
+* Flyway
+* Liquibase
 
 ## How to use?
 
-1. **With PostgreSQL and Flyway migrations**
+1. **With PostgreSQL and Flyway/Liquibase migrations**
 
 ```xml
+
 <project>
     <properties>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <testcontainers.version>1.18.1</testcontainers.version>
-        <jooq-testcontainers-codegen-maven-plugin.version>0.0.1</jooq-testcontainers-codegen-maven-plugin.version>
+        <jooq-testcontainers-codegen-maven-plugin.version>0.0.3-SNAPSHOT</jooq-testcontainers-codegen-maven-plugin.version>
```

Comment:
This should be 0.0.2

Reply:
Why? I upgraded main project version

Reply:
My Bad it should be 0.0.3, as when this code is merged to main branch SNAPSHOT should be discarded.